app/views/JourneyRecoveryStartAgainView.scala.html

```diff
@@ -30,7 +33,7 @@ <h1 class="govuk-heading-xl">@messages("journeyRecovery.title")</h1>
     <p class="govuk-body">
         @govukButton(
             ButtonViewModel(messages("site.startAgain"))
-                .asLink(routes.IndexController.onPageLoad().url)
+                .asLink(frontendAppConfig.sdilHomeUrl)
```

Comment:
The local application.conf value for **frontendAppConfig.sdilHomeUrl** is ".../account-home" but the actual route in the account frontend is ".../home" so it would not be found. 
It does look to be the correct url in other app-config-envs.

Please update the application.conf value

Reply:
<img width="701" alt="Screenshot 2024-04-23 at 10 13 35" src="https://github.com/hmrc/soft-drinks-industry-levy-returns-frontend/assets/13115186/0eb058b0-1313-4f65-a02d-cc41aa5a4a71">
Appears to be in returns-frontend? 🤔